CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
```

Comment:
I don't think diving straight into the issues would be a good idea.

It would be best if they first start by chatting with us, IMO linking https://web.libera.chat/#osrd would be better here